### ISSUE TITLE
iOS app rejected multiple times for button style on "Sign in with Apple"

### ISSUE THREAD

**Issue by Johnsgaard** [auth0/lock#1877]
### Description

I had a phone call with the Apple review team and the only thing that is preventing my app from getting onto the app store is a "gray" border on the "Sign in with Apple" button. This is happening with the "New" Universal form.

I have attempted a submission with BOTH Classic and Universal versions of the form with no customization other than icon and primary color / background color.

### Reproduction
<img width="750" alt="Screen Shot 2020-06-02 at 9 11 46 PM" src="https://user-images.githubusercontent.com/11530155/83591660-ae933280-a515-11ea-9472-12f8215a2023.png">

![Universal](https://user-images.githubusercontent.com/11530155/83591876-3711d300-a516-11ea-8f4c-cc5993a9d764.jpeg)

### Environment

- **Version of this library used:**
New Universal Login form
- **Version of the platform or framework used, if applicable:**
React-native Expo - sdk version 36.0.0
- **Other relevant versions (language, server software, OS, browser):**
iOS
- **Other modules/plugins/libraries that might be involved:**
lock

**Comment by Johnsgaard** [comment]
All the apple store requires is that the border of the button is black 😕

**Comment by stevehobbsdev** [comment]
Thanks @Johnsgaard, let me raise the issue with the new Universal Login Page with the relevant team internally.

As for Lock, the Apple button does look quite different - are you saying you've had a submission failure with Lock as well, and what kind of feedback did you receive about the failure?

![image](https://user-images.githubusercontent.com/766403/83642253-7e47a480-a5a6-11ea-84f9-687d96a8f32d.png)

**Comment by stevehobbsdev** [comment]
@Johnsgaard I see a separate issue [has been raised here](https://github.com/auth0/react-native-auth0/issues/310), so let's keep the discussion about the new ULP over there.

Here, I'd like to talk about Lock. If you can confirm what feedback you've received about the Apple button on the classic login page, we can start to help.

**Comment by Johnsgaard** [comment]
@stevehobbsdev I have been rejected with both Lock and Universal Lock. They didn't give me any reasoning on the `Lock` version other than the Apple button doesn't follow the guidelines. So I updated to Universal and gave that a shot. Also rejected on that one because of a border. At least I know now why the button is rejected and reached out to your team to see what can be done about the border.

**Comment by stevehobbsdev** [comment]
Thanks - let's leave the new ULP with that team; they are aware.

Can you let me know when you tried with Lock? Just so I can get a timeline. We did have a rejection early on but made some updates to resolve that. We've had confirmation from those customers that their apps were since accepted. We also know that getting something accepted can sometimes be down to luck as to who reviews your app. We believe the button we have on Lock _now_ is the best effort according to their guidance, but please let us know if you try again with Lock and manage to wrangle any more feedback from them.

**Comment by Johnsgaard** [comment]
No problem.

So the submission with `Lock` was first rejected because I didn't support a sign in with apple option.

<img width="1084" alt="Screen Shot 2020-06-03 at 8 31 23 AM" src="https://user-images.githubusercontent.com/11530155/83649740-c6e66a00-a574-11ea-9fda-539f3257cca8.png">

That submission was May 28th. So I added Sign in with apple support and fixed up some other issues they had (push notifications). After that was all done and another binary was submitted they gave me this rejection:

<img width="726" alt="Screen Shot 2020-06-03 at 8 30 28 AM" src="https://user-images.githubusercontent.com/11530155/83649881-eed5cd80-a574-11ea-8ce7-82fc05f990dd.png">

They didn't give me much context but all I can tell you is that I used the standard (Classic) Lock form with no custom HTML. Just added a custom icon and colors. I have a feeling they rejected it because of how the icon scales with the text?

I am going to try to re-submit with the `Lock` seeing as I talked to an Apple reviewer and she mentioned the outline is the only thing I need to worry about so if I can re-show her the Classic style I will see if that progresses me through the process.

**Comment by Johnsgaard** [comment]
![IMG_7577](https://user-images.githubusercontent.com/11530155/83650855-0feaee00-a576-11ea-82b7-2e6cb574ae91.PNG)

**Comment by stevehobbsdev** [comment]
Great - if it fails and you can get some more detailed feedback out of them, please let us know!

**Comment by Johnsgaard** [comment]
@stevehobbsdev The app was now approved with the `Lock v 11.24.1` (Screenshot) Thank you for taking the time to look at my issue.

**Comment by ntimko** [comment]
@stevehobbsdev - we are also using Lock 11.24.1 and have been rejected by Apple. Created support ticket and they referenced me back to here. Just thought I'd let it be know.

We have requested feedback from Apple Review and no response. Just for reference, we did post a reply back to the reviewer saying:

1) We use a black button on a white background to provide sufficient contrast - check
2) We say "Sign in with Apple" - conforming to the capitalization policy stating that only the words "Sign" and "Apple" should be capitalized - check
3) The text is 43% of the size of the overall height of the button - check
4) The text and logo are left aligned which is allowed when using multiple sign-in methods - check

Any guidance other than simply re-submit? Appreciate any and all support. Below is screenshot of what Apple rejected.

![appStore_rejection_signIn_2020-06-03](https://user-images.githubusercontent.com/63062053/83795273-72b6b500-a65c-11ea-86c6-3d74a3a42c3d.png)
.

**Comment by stevehobbsdev** [comment]
@ntimko Unfortunately I have no other advice other than to resubmit and see if you get another reviewer who is more lenient. We've had a few cases where customers have submitted exactly the same as you have and it's passed, and some have failed. I would love to hear any feedback you get from this as to why it was rejected with `11.24.1`.

**Comment by ntimko** [comment]
Thanks Steve. We re-submitted yesterday and will let you know the outcome along with any feedback we get.

**Comment by ntimko** [comment]
@stevehobbsdev we were rejected again using lock 11.24.1 (4th attempt) and the apple reviewer also said they had wifi issues when trying to go through the sign in with apple work flow (3rd time we got that, appears to be fine on our side).

End of the day we have made the decision to remove social logins as we dont seem to be making any progress with apple and all rejections we have received related to "Sign in with Apple" button and/or workflow. Unfortunate but need to move forward.
